test/routable_component.js

```diff
@@ -34,22 +34,22 @@ class MyController {
 
 /** @test {RoutableComponent} */
 describe('RoutableComponent', function() {
-  lazy('routes', function() { return new RoutableComponentRoutes(this.routing_classes) });
-  lazy('routing_classes', function() { return MyUsefulRouting });
+  lazy('routes', function() { return new RoutableComponentRoutes(this.routingClasses) });
```

Comment:
__[eslint]__
* Error - 'lazy' is not defined. (no-undef)
* Error - Unexpected 'this'. (no-invalid-this)
* Error - Missing semicolon. (semi)

<!-- Comment by [SideCI](https://sideci.com) -->